src/Sql/Sql/Database Backup/Cmdlet/RestoreAzureRMSqlDatabase.cs

```diff
@@ -295,6 +294,14 @@ public class RestoreAzureRmSqlDatabase
             HelpMessage = "The zone redundancy to associate with the Azure Sql Database. This property is only settable for Hyperscale edition databases.")]
         public SwitchParameter ZoneRedundant { get; set; }
 
+        /// <summary>
+        /// Gets or sets the tags associated with the Azure Sql Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The tags to associate with the Azure Sql Database Server")]
```

Comment:
Remove "Server" from the HelpMessage: i.e.
"The tags to associate with the Azure Sql Database"